wqflask/wqflask/correlation_matrix/show_corr_matrix.py

```diff
@@ -165,16 +167,13 @@ def __init__(self, start_vars):
 
         self.pca_works = "False"
         try:
-            corr_result_eigen = np.linalg.eig(np.array(self.pca_corr_results))
-            corr_eigen_value, corr_eigen_vectors = sortEigenVectors(
-                corr_result_eigen)
 
             if self.do_PCA == True:
                 self.pca_works = "True"
                 self.pca_trait_ids = []
-                pca = self.calculate_pca(
-                    list(range(len(self.traits))), corr_eigen_value, corr_eigen_vectors)
-                self.loadings_array = self.process_loadings()
+                pca = self.calculate_pca()
+                self.loadings_array = process_factor_loadings_tdata(
+                    self.loadings, len(self.trait_list))
```

Comment:
**nitpick**: for readability issues, it's usually nice to specify keywords as: `my_func(a=a, b=b)`.